code/game/objects/effects/aliens.dm

```diff
@@ -69,6 +75,10 @@
 	
 	if(TIMER_COOLDOWN_CHECK(src, COOLDOWN_ACID))
 		return
+	
+	if(world.time <= acid_spray.creation_time + 5) //To prevent being able to walk "over" acid sprays
+		acid_spray_act(acid_spray.xeno_owner)
+		return
```

Comment:
Why isnt this a cooldown timer or something

Reply:
why should it be? a cooldown timer for that kind of thing is just not optimal. 
Why calling a timer on all the tiles when you can just check a simple operation when a human walk on it

Reply:
No I mean you start the timer on creation and then just have COOLDOWN checks here